aten/src/ATen/core/interned_strings.h

```diff
@@ -246,6 +246,10 @@ struct CAFFE2_API Symbol {
   bool is_prim() const;
   bool is_onnx() const;
   bool is_user() const;
+  // Does this symbol live in one of the above built-in namespaces?
+  // If not, that means that this symbol came from the user, through the
+  // defintiion of a custom operator.
+  bool is_builtin_ns() const;
```

Comment:
Please do not commit this. I do not want to have people be able to easily special case stuff for built in ops. This will create huge long-term tech debt even if commit for a little while.